GVFS/GVFS/CommandLine/IncludeVerb.cs

```diff
@@ -0,0 +1,182 @@
+
+        [Option(
+            'a',
+            "add",
+            Required = false,
+            Default = "",
+            HelpText = "A semicolon-delimited list of folders to include. Wildcards are not supported.")]
+        public string Add { get; set; }
+
+        [Option(
+            'r',
+            "remove",
+            Required = false,
+            Default = "",
+            HelpText = "A semicolon-delimited list of folders to remove for being included. Wildcards are not supported.")]
+        public string Remove { get; set; }
+
+        [Option(
+            'l',
+            "list",
+            Required = false,
+            Default = false,
+            HelpText = "List of folders to for being included in projection.")]
+        public bool List { get; set; }
+
+        protected override string VerbName => IncludeVerbName;
+
+        protected override void Execute(GVFSEnlistment enlistment)
+        {
+            using (JsonTracer tracer = new JsonTracer(GVFSConstants.GVFSEtwProviderName, "Include"))
+            {
+                try
+                {
+                    tracer.AddLogFileEventListener(
+                        GVFSEnlistment.GetNewGVFSLogFileName(enlistment.GVFSLogsRoot, GVFSConstants.LogFileTypes.Include),
+                        EventLevel.Informational,
+                        Keywords.Any);
+
+                    using (GVFSDatabase database = new GVFSDatabase(new PhysicalFileSystem(), enlistment.EnlistmentRoot, new SqliteDatabase()))
+                    {
+                        IncludedFolderTable includedFolderTable = new IncludedFolderTable(database);
+                        bool haveFoldersToAdd = !string.IsNullOrEmpty(this.Add);
+                        bool haveFoldersToRemove = !string.IsNullOrEmpty(this.Remove);
+
+                        if (this.List || (!haveFoldersToAdd && !haveFoldersToRemove))
+                        {
+                            List<string> directories = includedFolderTable.GetAll();
+                            if (directories.Count == 0)
+                            {
+                                this.Output.WriteLine("No folders in included list.");
+                            }
+                            else
+                            {
+                                foreach (string directory in directories)
+                                {
+                                    this.Output.WriteLine(directory);
+                                }
+                            }
+
+                            return;
+                        }
+
+                        // Make sure there is a clean git status before allowing inclusions to change
+                        this.CheckGitStatus(tracer, enlistment);
+
+                        if (haveFoldersToRemove)
+                        {
+                            foreach (string directoryPath in this.Remove.Split(new[] { ';' }, StringSplitOptions.RemoveEmptyEntries))

```

Comment:
I will add to the help text about being relative to the repo root and we can get feedback to see if we need to do more than that.

BTW `GVFS` and `GVFS\` should be considered the same because the code that creates the `IncludedFolderData` does this:
```
string[] folders = directoryPath.Split(new[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar }, StringSplitOptions.RemoveEmptyEntries);
```

Reply:
> BTW GVFS and GVFS\ should be considered the same because the code that creates the IncludedFolderData does this:

Ok thanks I missed this!